buildout.cfg

```diff
@@ -18,6 +18,9 @@ parts +=
     rebuild_i18n-sh
     robot
 
+test-eggs +=
```

Comment:
@rodfersou this should be added on a version basis also or we will end with the package installed in Plone 5